pkg/workflows/workload/create.go

```diff
@@ -0,0 +1,59 @@
+package workload
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/filewriter"
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/providers"
+	"github.com/aws/eks-anywhere/pkg/task"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+// Create is a schema for create cluster.
+type Create struct {
+	provider         providers.Provider
+	clusterManager   interfaces.ClusterManager
+	gitOpsManager    interfaces.GitOpsManager
+	writer           filewriter.FileWriter
+	eksdInstaller    interfaces.EksdInstaller
+	clusterCreator   interfaces.ClusterCreator
+	packageInstaller interfaces.PackageInstaller
+}
+
+// NewCreate builds a new create construct.
+func NewCreate(provider providers.Provider,
+	clusterManager interfaces.ClusterManager, gitOpsManager interfaces.GitOpsManager,
+	writer filewriter.FileWriter,
+	clusterCreator interfaces.ClusterCreator,
+	eksdInstaller interfaces.EksdInstaller,
+	packageInstaller interfaces.PackageInstaller,
+) *Create {
+	return &Create{
+		provider:         provider,
+		clusterManager:   clusterManager,
+		gitOpsManager:    gitOpsManager,
+		writer:           writer,
+		eksdInstaller:    eksdInstaller,
+		clusterCreator:   clusterCreator,
+		packageInstaller: packageInstaller,
+	}
+}
+
+// Run Create implements create functionality for workload cluster's create operation.
```

Comment:
nit: this comment seems a bit convoluted?
```suggestion
// Run executes the tasks to create a workload cluster.
```